package.json

```
@@ -53,7 +53,9 @@
     "install": "prebuild-install -r napi || cmake-js rebuild",
     "rebuild": "npm run rebuild-changes",
     "build": "npm run build-changes",
+    "build-arm": "npm run build-changes-arm",
```

```suggestion
    "build:arm": "npm run build-changes-arm",
```
Its pretty common (and we use this in other packages) to use a comma to indicate a "variant" / sub-command.

---

It would be great to get this included in the thoughts for https://github.com/realm/realm-js/issues/3708